app/src/main/java/org/oppia/app/player/state/StateAdapter.kt

```diff
@@ -138,6 +156,7 @@ class StateAdapter(
       is NumericInputInteractionViewModel -> VIEW_TYPE_NUMERIC_INPUT_INTERACTION
```

Comment:
Your PR should not have any implemenation of `NumericInputInteractionViewModel` and `TextInputInteractionViewModel`

Reply:
It is from the base PR, if I remove it now it will show changes in file.